packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -947,16 +937,8 @@ module('Integration | operator-mode', function (hooks) {
           ?.textContent?.trim() == 'Saving…',
       { timeoutMessage: 'Waiting for Saving... to appear' },
     );
-    await waitUntil(
-      () =>
-        document
-          .querySelector('[data-test-auto-save-indicator]')
-          ?.textContent?.trim() == 'Failed to save: Rejected by firewall',
-      { timeoutMessage: 'Waiting for "Failed to save" to appear' },
-    );
-    assert
-      .dom('[data-test-auto-save-indicator]')
-      .containsText('Failed to save: Rejected by firewall');
```

Comment:
This is another example of a live transition into an error state since the identity map replaces the running instance with the error